pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/Language.kt

```diff
@@ -25,8 +25,8 @@ sealed class Language {
 
     fun locale(): Locale =
         when (this) {
-            Bokmal -> Locale.forLanguageTag("NB")
-            Nynorsk -> Locale.forLanguageTag("NN")
+            Bokmal -> Locale.forLanguageTag("no")
+            Nynorsk -> Locale.forLanguageTag("no")
```

Comment:
Blir dette samme for Nynorsk og Bokmål?

Reply:
Ja. Locale er brukt for dato formattering, kommatall, kroneverdier osv. Det var en feil med at NN ikke hadde noen form for formattering ettersom det kun lå på "no" locale.